### PR TITLE
fix: fix java okhttp snippet

### PR DESCRIPTION
**Pull request by lstocchi**
### What does this PR do?

This PR enhance the JAVA okhttp snippet

### Screenshot / video of UI

java_okhttp_snippet

### What issues does this PR fix or reference?

it is part of #404 

### How to test this PR?

1. inference server -> java -> okhttp -> you should see the new snippet with dep info and code

### CODE REVIEW

File: packages/backend/src/templates/java-okhttp.mustache [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import com.squareup.okhttp.MediaType;
+import com.squareup.okhttp.OkHttpClient;
+import com.squareup.okhttp.Request;
+import com.squareup.okhttp.RequestBody;
+import com.squareup.okhttp.Response;
+
+OkHttpClient client = new OkHttpClient();
+MediaType mediaType = MediaType.parse("application/json");
+RequestBody body = RequestBody.create(mediaType, "{\n  \"messages\": [\n    {\n      \"content\": \"You are a helpful assistant.\",\n      \"role\": \"system\"\n    },\n    {\n      \"content\": \"What is the capital of France?\",\n      \"role\": \"user\"\n    }\n  ]\n}");
```
Comment (axel7083) [inline packages/backend/src/templates/java-okhttp.mustache:21]: Nice having more variants !

Just one question: could we improve the body to have it a better look ? Like multi string,?

File: packages/backend/src/templates/java-okhttp.mustache [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Request request = new Request.Builder()
+.url("http://localhost:53891/v1/chat/completions")
```
Comment (jeffmaury) [inline packages/backend/src/templates/java-okhttp.mustache:23]: Why is the URL hardcoded ? It should use the data from the Request options

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: packages/backend/src/templates/java-okhttp.mustache (hunk 2 above)
Comment (lstocchi) [reply packages/backend/src/templates/java-okhttp.mustache:23]: good catch, fixing

**Review by benoitf — APPROVED** [review]

File: packages/backend/src/templates/java-okhttp.mustache [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+MediaType mediaType = MediaType.parse("application/json");
+RequestBody body = RequestBody.create(mediaType, "{\n  \"messages\": [\n    {\n      \"content\": \"You are a helpful assistant.\",\n      \"role\": \"system\"\n    },\n    {\n      \"content\": \"What is the capital of France?\",\n      \"role\": \"user\"\n    }\n  ]\n}");
+Request request = new Request.Builder()
+.url("{{{ endpoint }}}")
+.method("POST", body)
+.addHeader("Content-Type", "application/json")
+.build();
+Response response = client.newCall(request).execute();
+
+======
```
Comment (benoitf) [inline packages/backend/src/templates/java-okhttp.mustache:29]: missing EOF newline

**Review by jeffmaury — APPROVED** [review]
LGTM. Multi line would be a plus

**Pull request merged**